net-p2p/go-ethereum/go-ethereum-1.4.16.ebuild

```diff
@@ -0,0 +1,42 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+inherit eutils
+
+DESCRIPTION="Official golang implementation of the Ethereum protocol"
+HOMEPAGE="https://github.com/ethereum/go-ethereum"
+SRC_URI="https://github.com/ethereum/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3+ LGPL-3+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="evm"
+
+DEPEND="
+	dev-lang/go
+"
+RDEPEND="${DEPEND}"
+
+src_prepare() {
+	sed -i \
+		-e 's|\$(GOROOT)/bin/go|go|g' \
+		Makefile
+
+	default
+}
+
+src_compile() {
+	emake geth
+	use evm && emake evm
+}
+
+src_install() {
+	for d in README* COPYING* AUTHORS; do
+		[[ -s "${d}" ]] && dodoc "${d}"
+	done
```

Comment:
get rid of this whole for loop, and in its place just put `einstalldocs` (screw the `COPYING*` stuff)